starknet_py/serialization/data_serializers/option_serializer.py

```diff
@@ -1,11 +1,13 @@
 from dataclasses import dataclass
 from typing import Any, Generator, Optional
 
-from starknet_py.serialization import CairoDataSerializer
 from starknet_py.serialization._context import (
     DeserializationContext,
     SerializationContext,
 )
+from starknet_py.serialization.data_serializers.cairo_data_serializer import (
```

Comment:
What was the exact problem that required this change?

Reply:
because if it is imported from `starknet_py.serialization` it create `circular imports`